angular.json

```diff
@@ -1,47 +1,49 @@
 {
-  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
-  "version": 1,
-  "newProjectRoot": ".",
-  "projects": {
-    "angularfire": {
-      "projectType": "library",
-      "root": "src",
-      "sourceRoot": "src",
-      "prefix": "angularfire",
-      "architect": {
-	"build": {
-	  "builder": "@angular-devkit/build-angular:ng-packagr",
-	  "options": {
-	    "tsConfig": "tsconfig.json",
-	    "project": "src/package.json"
-	  }
+	"$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+	"version": 1,
+	"newProjectRoot": ".",
+	"projects": {
+		"angularfire": {
+			"projectType": "library",
+			"root": "src",
+			"sourceRoot": "src",
+			"prefix": "angularfire",
+			"architect": {
+				"build": {
+					"builder": "@angular-devkit/build-angular:ng-packagr",
+					"options": {
+						"tsConfig": "tsconfig.json",
+						"project": "src/ng-package.json"
+					}
+				},
+				"test": {
+					"builder": "@angular-devkit/build-angular:karma",
+					"options": {
+						"polyfills": [
+              "zone.js",
+              "zone.js/testing"
+            ],
+						"tsConfig": "tsconfig.spec.json",
+						"karmaConfig": "karma.conf.js"
+					}
+				},
+				"lint": {
+					"builder": "@angular-devkit/build-angular:tslint",
+					"options": {
+						"tsConfig": [
+							"tsconfig.json",
+							"tsconfig.spec.json"
+						],
+						"exclude": [
+							"**/node_modules/**"
+						]
+					}
+				}
+			}
+		}
 	},
-	"test": {
-	  "builder": "@angular-devkit/build-angular:karma",
-	  "options": {
-	    "main": "src/test.ts",
-	    "tsConfig": "tsconfig.spec.json",
-	    "karmaConfig": "karma.conf.js"
-	  }
-	},
-	"lint": {
-	  "builder": "@angular-devkit/build-angular:tslint",
-	  "options": {
-	    "tsConfig": [
-	      "tsconfig.json",
-	      "tsconfig.spec.json"
-	    ],
-	    "exclude": [
-	      "**/node_modules/**"
-	    ]
-          }
-        }
-      }
-    }
-  },
-  "defaultProject": "angularfire",
-  "cli": {
-	"packageManager": "yarn",
-    "analytics": "86795b8f-9036-4a53-929c-a7303453d677"
-  }
+	"cli": {
+		"packageManager": "yarn",
```

Comment:
is it still true that the package manager is yarn?

Reply:
Yep! We could migrate off but no need to right now.